queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```

Comment:
- [ ] Here, the date format is not correct.  Instead of '2019-12-31', it should be '2019-31-12', as stated by the previous reviewer.  Please, kindly correct this date properly and we will be ready for approval 💪

Reply:
@Meltrust That should be an error from the previous reviewer which I corrected. The [appropriate date format is YY-MM-DD.](https://www.postgresql.org/docs/9.1/functions-formatting.html) or [DD-MM-YY.](https://sqlserverguides.com/postgresql-date-format/). When entering data, I have used the format `YY-MM-DD` and so any query that breaks this format is regarded as an error. 

Kindly run the query you suggest to confirm.

Thanks.

Reply:
_Note for the next reviewer:  The student is right because he is using the YY-MM-DD format.  The last reviewer and therefore, me too, requested an incorrect format.  I'll edit this requirement to `optional`.  The student does not have to implement it._